ax25-apps fails to build on Debian Trixie

For the current top of tree which is version 2.0.1, I see the following failure:

configure.ac:15: the top level
   dh_auto_configure
	./configure --build=aarch64-linux-gnu --prefix=/usr --includedir=\${prefix}/include --mandir=\${prefix}/share/man --infodir=\${prefix}/share/info --sysconfdir=/etc --localstatedir=/var --disable-option-checking --disable-silent-rules --libdir=\${prefix}/lib/aarch64-linux-gnu --libexecdir=\${prefix}/lib/aarch64-linux-gnu --disable-maintainer-mode --disable-dependency-tracking
checking for a BSD-compatible install... /usr/bin/install -c
checking whether sleep supports fractional seconds... yes
checking filesystem timestamp resolution... 0.01
checking whether build environment is sane... yes
checking for a race-free mkdir -p... /usr/bin/mkdir -p
checking for gawk... no
checking for mawk... mawk
checking whether make sets $(MAKE)... yes
checking whether make supports nested variables... yes
checking xargs -n works... yes
checking for gcc... gcc
checking whether the C compiler works... yes
checking for C compiler default output file name... a.out
checking for suffix of executables... 
checking whether we are cross compiling... no
checking for suffix of object files... o
checking whether the compiler supports GNU C... yes
checking whether gcc accepts -g... yes
checking for gcc option to enable C11 features... none needed
checking whether gcc understands -c and -o together... yes
checking whether make supports the include directive... yes (GNU style)
checking dependency style of gcc... none
checking for ax25_config_load_ports in -lax25... yes
checking for initscr in -lncursesw... yes
checking for dirent.h that defines DIR... yes
checking for library containing opendir... none required
checking for sys/wait.h that is POSIX.1 compatible... yes
checking for stdio.h... yes
checking for stdlib.h... yes
checking for string.h... yes
checking for inttypes.h... yes
checking for stdint.h... yes
checking for strings.h... yes
checking for sys/stat.h... yes
checking for sys/types.h... yes
checking for unistd.h... yes
checking for vfork.h... no
checking for sys/time.h... yes
checking for sys/select.h... yes
checking for sys/socket.h... yes
checking for utime.h... yes
./configure: line 5010: #include: command not found
checking for arpa/inet.h... yes
checking for fcntl.h... yes
checking for limits.h... yes
checking for memory.h... yes
checking for netdb.h... yes
checking for netinet/in.h... yes
checking for sys/ioctl.h... yes
checking for sys/socket.h... (cached) yes
checking for sys/time.h... (cached) yes
checking for syslog.h... yes
checking for termio.h... yes
checking for termios.h... yes
checking for utime.h... (cached) yes
checking for ncursesw/curses.h... yes
checking for an ANSI C-conforming const... yes
checking for inline... inline
checking for off_t... yes
checking whether struct tm is in sys/time.h or time.h... time.h
checking for working volatile... yes
checking for pid_t... yes
checking for fork... yes
checking for vfork... yes
checking for alarm... yes
checking for working fork... yes
checking for working vfork... (cached) yes
checking build system type... aarch64-unknown-linux-gnu
checking host system type... aarch64-unknown-linux-gnu
checking for GNU libc compatible malloc... yes
checking for working memcmp... yes
checking for working mktime... yes
checking types of arguments for select... int,fd_set *,struct timeval *
checking whether lstat correctly handles trailing slash... yes
checking whether stat accepts an empty string... no
checking whether utime accepts a null argument... yes
checking for bzero... yes
checking for gethostbyname... yes
checking for inet_ntoa... yes
checking for memchr... yes
checking for memmove... yes
checking for memset... yes
checking for select... yes
checking for socket... yes
checking for strcasecmp... yes
checking for strchr... yes
checking for strdup... yes
checking for strerror... yes
checking for strrchr... yes
checking for strtol... yes
checking for strtoul... yes
checking for utime... yes
checking that generated files are newer than configure... done
configure: creating ./config.status
config.status: creating call/Makefile
config.status: creating ax25ipd/Makefile
config.status: creating listen/Makefile
config.status: creating Makefile
config.status: creating ax25rtd/Makefile
config.status: creating ax25mond/Makefile
config.status: creating doc/Makefile
config.status: creating man/Makefile
config.status: creating etc/Makefile
config.status: creating ax25apps.spec
config.status: creating config.h
config.status: executing depfiles commands
   dh_auto_build
	make -j4
.
.
.
listener.c: In function ‘ax25_receive’:
listener.c:405:17: error: implicit declaration of function ‘time’ [-Wimplicit-function-declaration]
  405 |         stamp = time(NULL);
      |                 ^~~~
listener.c:47:1: note: ‘time’ is defined in header ‘<time.h>’; this is probably fixable by adding ‘#include <time.h>’

--- a/ax25apps/ax25rtd/listener.c
+++ b/ax25apps/ax25rtd/listener.c
@@ -29,9 +29,7 @@
 #include <unistd.h>
 #include <signal.h>
 #include <sys/ioctl.h>
-#ifdef TIME_WITH_SYS_TIME
 #include <time.h>
-#endif
 #include <sys/time.h>
 #include <netinet/in.h>
 #include <net/route.h>

